command/agent/consul/connect.go

```diff
@@ -95,6 +95,18 @@ func newConnectGateway(connect *structs.ConsulConnect) *api.AgentServiceConnectP
 	return &api.AgentServiceConnectProxyConfig{Config: envoyConfig}
 }
 
+// newWeights creates a new Consul AgentWeights struct based on a Nomad ConsulWeights struct.
+func newWeights(weights *structs.ConsulWeights) *api.AgentWeights {
+	if weights == nil {
+		return nil
+	}
+
+	return &api.AgentWeights{
+		Passing: weights.Passing,
+		Warning: weights.Warning,
+	}
+}
```

Comment:
This function is only used in one place and it's being defined in an unrelated file. Let's move this to where it's being used in `service_client.go`.